### ISSUE TITLE
TestOutputOrganizerTest failed; need to consider Windows-style path representation

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/unittest-helper#28]
0.3.2-SNAPSHOT, several tests failed.

## TestOutputOrganizerTest#test_toHomeRelativeString_simple
```
org.opentest4j.AssertionFailedError: 
expected: "~/github/unittest-helper/lib"
 but was: "~/github\unittest-helper\lib"
	at java.base@17.0.8/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base@17.0.8/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base@17.0.8/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base@17.0.8/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at app//com.kazurayam.unittest.TestOutputOrganizerTest.test_toHomeRelativeString_simple(TestOutputOrganizerTest.java:124)
	...
```

## TestOutputOrganizerTest#test_toHomeRelativeString_simple_more
```
org.opentest4j.AssertionFailedError: 
expected: "~/github/unittest-helper/lib/test-output/foo.txt"
 but was: "~/github\unittest-helper\lib\test-output\foo.txt"
	at java.base@17.0.8/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base@17.0.8/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base@17.0.8/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base@17.0.8/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at app//com.kazurayam.unittest.TestOutputOrganizerTest.test_toHomeRelativeString_simple_more(TestOutputOrganizerTest.java:132)
	...
```

## TestOutputOrganizerTest#test_toHomeRelativeString_when_not_relative
```
org.opentest4j.AssertionFailedError: 
expected: "/Applications"
 but was: "C:\Applications"
	at java.base@17.0.8/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base@17.0.8/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base@17.0.8/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base@17.0.8/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at app//com.kazurayam.unittest.TestOutputOrganizerTest.test_toHomeRelativeString_when_not_relative(TestOutputOrganizerTest.java:149)
    ...
```

**Comment by kazurayam** [comment]
I need to decide how the `toHomeRelativeString()` should work.

Option1: even on Windows, it should return "~/github/unittest-helper/lib", rather than "~\github\unittest-helper\lib"
Option2: on Windows, it should return "~\github\unittest^helper\lib", rather than "~/github/unittest-helper/lib"

Which way to take?

...

I would take the Option 1.

**Comment by kazurayam** [comment]
What to do with the case of absoluth path?

On Windows, if I have a folder `C:\Application`, then what value should the `TestOutputOgainzer#toHomeRelativeString()` method return?

**Comment by kazurayam** [comment]
fixed at v0.3.3
